docs/UserGuide.md

```diff
@@ -486,6 +486,15 @@ Paidlancers data are saved in the hard disk automatically on command issue. Ther
 |   [View Upcoming Events](#view-upcoming-events)    |                                 `remind DAYS`                                 |                                                      `remind 2`                                                       |
 |              [Save data](#save-data)               |                                      NIL                                      |                                                          NIL                                                          |
 
+Thank you for taking the time to read through this user guide:smile:! We hope that it has provided you with the
+information you need to make the most out of Paidlancers!:innocent:
```

Comment:
Would it look nicer to add a space before every emoji?

Reply:
Fixed!